extensions/mssql/src/sqlToolsServer.ts

```diff
@@ -58,13 +58,13 @@ export class SqlToolsServer {
 				});
 			});
 			statusView.show();
-			statusView.text = 'Starting service';
+			statusView.text = localize('startingServiceStatusMsg', "Starting service");
 			this.client.start();
 			await this.activateFeatures(context);
 			return this.client;
 		} catch (e) {
 			Telemetry.sendTelemetryEvent('ServiceInitializingFailed');
-			vscode.window.showErrorMessage('Failed to start Sql tools service');
+			vscode.window.showErrorMessage(localize('failedToStartServiceErrorMsg', "Failed to start Sql tools service"));
```

Comment:
you may change `Sql tools service` to `SQL Tools Service` since you're already making updates here.